unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/StringIgnoreCaseEqualityComparer.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+
+namespace DCL
+{
+    public class StringIgnoreCaseEqualityComparer : IEqualityComparer<string>
```

Comment:
Are you sure we didn't have already an IEqualityComparer for the ignore case? I remember misha talking about it but no clue on where to find it xD